CHANGELOG.md

```diff
@@ -37,13 +44,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   issues with L025 and L026 with BigQuery column references involving `STRUCT`.
 - Fix bug [#1080](https://github.com/sqlfluff/sqlfluff/issues/1080), add
   SET SCHEMA and DROP SCHEMA support to ANSI dialect.
-- Added support for additional delimiters by creating a new DelimiterSegment in the 
-  ANSI dialect which defaults to the semicolon, but allows it to be more intuitive 
+- Added support for additional delimiters by creating a new DelimiterSegment in the
+  ANSI dialect which defaults to the semicolon, but allows it to be more intuitive
   when overriden in a specific child dialect (mysql) [#901](https://github.com/sqlfluff/sqlfluff/issues/901))
 
-Contributors:
-- [@bolajiwahab](https://github.com/bolajiwahab) ([#1063])(https://github.com/sqlfluff/sqlfluff/pull/1063)
-
```

Comment:
I think we should either keep this bit - or reference the user in the but that they changed. In this case that would be _Allow `CREATE FUNCTION` syntax without arguments_.

I realise we haven't been very consistent about using the contributors section, but we should either use it or just remove it.

Reply:
Yeah it felt a bit silly only referencing one contributor. I'll just mention them in the feature added

Reply:
Can you approve for when it's sorted? Then I can get this merged